shopfloor_reception/services/reception.py

```diff
@@ -766,25 +766,23 @@ def _align_product_uom_qties(self, move):
         # If move is already done, do not update lines qties
         # if move.state in ("done", "cancel"):
         #     return
-
+        move = line.move_id
         qty_todo = move.product_uom_qty
```

Comment:
Compute all in line uom
```suggestion
        qty_todo = self.env['uom.uom'].browse(move.product_uom_id)._compute_quantity(move.product_uom_qty, move.product_uom_id, round=False)
```